inference-engine/src/mkldnn_plugin/utils/shape_inference/static_dimension.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+#include <stdexcept>
+#include <ostream>
+
+namespace ov {
+/// \brief Class representing a dimension, which must be static,
+///        in a shape or shape-like object.
+///
+/// Provides similar API to the public Dimension class.
+class StaticDimension {
```

Comment:
I don't see any tests for `StaticDimension`

Reply:
@jane-intel what is about tests? :)